core/common/assets/js/components/component-modal.js

```diff
@@ -14,6 +14,20 @@ export default class extends Component {
 		} );
 	}
 
```

Comment:
In this file, consider using `forceMethodImplementation` instead of throwing error inside `getModalLayout` method